recipes-bsp/optee-os/optee-os-iot2050_3.16.0_rpmbw.bb

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Ivan Mikhaylov <ivan.mikhaylov@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+require optee-os-iot2050_3.16.0.inc
+
+OPTEE_EXTRA_BUILDARGS += " CFG_RPMB_WRITE_KEY=y"
+DEFAULT_PREFERENCE = "-1"
```

Comment:
Where is this used?

Reply:
It is for default preference for optee-os-iot2050_3.16.0.bb, it possible to use PREFERRED_PROVIDER instead of this option.

Reply:
This is first of all a bitbake variable, and at that level no one is using it. If it has a meaning for OP-TEE, you need to bring it into its (Debian) build environment. That maybe require a Debian build profile here to be future-proof.